caffe2/ideep/operators/transpose_op.cc

```diff
@@ -0,0 +1,35 @@
+#include <caffe2/ideep/ideep_utils.h>
+
+using namespace caffe2;
+
+namespace {
+
+class IDEEPTransposeOp final : public IDEEPOperator {
+ public:
+  USE_IDEEP_DEF_ALIASES();
+  USE_IDEEP_OPERATOR_FUNCTIONS();
+
+  IDEEPTransposeOp(const OperatorDef& operator_def, Workspace* ws)
+      : IDEEPOperator(operator_def, ws),
+        axes_(this->template GetRepeatedArgument<int>("axes")){ }
```

Comment:
When passing an empty vector to `transpose_from`, what does ideep do?
I'm asking because if `axes` attribute is not provided, caffe2 Transpose's semantics is to reverse the order https://caffe2.ai/docs/operators-catalogue.html#transpose.

Reply:
The semantics of axes in ideep is totally identical to that of caffe2.